packages/graphql-client/src/utils/graphql.ts

```diff
@@ -0,0 +1,76 @@
+import { clearUndefined, ensureDefaults } from "@fleet-sdk/common";
+
+const OP_NAME_REGEX = /(query|mutation)\s?([\w\-_]+)?/;
+export const DEFAULT_HEADERS: Headers = {
+  "content-type": "application/json; charset=utf-8",
+  accept: "application/graphql-response+json, application/json"
+};
+
+type Variables = Record<string, unknown> | null;
+type GraphQLOperation<R, V extends Variables> = (variables?: V) => Promise<GraphQLResponse<R>>;
+type Credentials = RequestCredentials;
+type Headers = HeadersInit;
+type Fetcher = typeof fetch;
+
+export interface GraphQLResponse<T> {
+  data?: T;
+}
+
+export interface ResponseParser {
+  parse(text: string): unknown;
+  stringify(value: unknown): string;
+}
+
+export interface RequestParams {
+  operationName?: string | null;
```

Comment:
Yes, because sometimes we need to explicitly set some variable to be equal to `null`.

Example: `{ R4: "deadbeef", R5: null }`, this will explicitly say to GraphQL server that I want all boxes which `R4` is equal to `deadbeef` and empty `R5`.